### ISSUE TITLE
Switch Contacts backend

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/maps#135]
With 9.0 the Contacts/CardDAV backend moved into core. So for the »show Contacts on map« feature to work, we need to switch the endpoint around. @v1r0x 

@DeepDiver1975 @Henni what needs to be done?

Also then for the markers we can show the new color+letter placeholders which @skjnldsv updated: https://github.com/owncloud/core/pull/23638

**Comment by v1r0x** [comment]
Should we drop OC8 support or try to use the new contacts backend and fallback to the old one if the new doesn't exist?

**Comment by jancborchardt** [comment]
I’d say dropping ownCloud 8.* support is totally fine. The app is in very early stages, so any time spent for backwards compatibility is wasted atm.

**Comment by skjnldsv** [comment]
Agreed :)

**Comment by v1r0x** [comment]
Ok, then we (in this case we means @Henni :D) should use the new backend and drop oc8 support.
I think (or rather hope) that we only have to fix these lines:
https://github.com/owncloud/maps/blob/master/js/script.js#L1100
https://github.com/owncloud/maps/blob/master/js/script.js#L1110
https://github.com/owncloud/maps/blob/master/js/script.js#L1291
https://github.com/owncloud/maps/blob/master/js/script.js#L1589-L1594

**Comment by e-alfred** [comment]
This would be quite a nice fix, because contacts now do not work on OC 9.0 at all.

**Comment by v1r0x** [comment]
Absolutely! I think @Henni (new contacts app dev) has a lot of work with the said app, but maybe he knows what to do?
